snappy/overlord.go

```diff
@@ -381,8 +381,8 @@ func (o *Overlord) partsForGlobExpr(globExpr string) (parts []*Snap, err error)
 			continue
 		}
 
-		origin, _ := originFromYamlPath(realpath)
-		snap, err := NewInstalledSnap(realpath, origin)
+		developer, _ := developerFromYamlPath(realpath)
```

Comment:
Indeed, they will go away naturally because the "developer" will no longer be used in the path.